data_structures.rst

```diff
@@ -1,38 +1,33 @@
 General API
 ===========
 
-  Functions and classes used generally across the interface.
+  Functions and classes used across the interface.
 
 Procedures
 ----------
 
 .. code:: python
 
     """
-    This code describes "generic" beamline procedures.
```

Comment:
Points in line 12-15 describe the way beamline-specific functions and paramereters should work. Are these points accepted? rejected? Left in limbo?

Reply:
I agree that we are missing a section on how beamline procedures are intended to work and what they are, but this only defines the data structure.  So we should maybe expand a bit on this and move the text more to the beamline.rst file.

I think we agree with the definition of ProcedureData, I at least nothing more to say about it for the moment.

Reply:
Moving the (modified) text sound like a good idea. But I am again a little confused by what goes where. Procedures are not inherently limited and could be used for (beamline-specific) functionality in any area: diffractometer, lims, workflow, ... The beamline file I thouight was limited to things that do not belong elsewhere (i.e. not diffractometer, not lims, not workflow, ...). So is this use of procedure data limited to a specific component, or is beamine.rst not a component but a sort of top-level, covers-everything file?

Reply:
In this way I think you are right, I think its fine to keep it like it is right now. By the way I added back the text basically as it were, but we just dont see it in this comment.